demos/CMakeLists.txt

```diff
@@ -8,6 +8,9 @@ project(Demos)
 
 option(ENABLE_PYTHON "Whether to build extension modules for Python demos" OFF)
 
+set(DEMOS "ALL" CACHE STRING "Specify demo to build using comma(ex. -DDEMOS=classification_demo,text_detection_demo). Default is ALL")
```

Comment:
I don't understand, what's the purpose of this option?

Reply:
Reworked as @Wovchena suggested.